src/less/components/action-list.less

```diff
@@ -0,0 +1,90 @@
+.action-list {
+    position: relative;
+    width: 1rem;
+
+    > .material-icons {
+        position: relative;
+        z-index: @zindex-action-list-anchor;
+        color: @gray-5;
+        line-height: 1.1;
+
+        &:hover {
+            color: @gray-2;
+            cursor: pointer;
+        }
+    }
+
+    .action-menu {
+        position: absolute;
+        top: -0.5rem;
+        right: -0.65rem;
+        z-index: @zindex-action-list;
+        height: 0;
+
+        > a {
+            display: flex;
+            padding: 0.3rem 3rem 0.3rem 0.5rem;
+            color: @gray-5;
+            visibility: hidden;
+            position: relative;
+
+            > .material-icons {
+                margin-right: 0.5rem;
+            }
+
+            &:active,
+            &:focus {
+                text-decoration: none;
+            }
+
+            &:hover {
+                text-decoration: none;
+                background-color: @gray-10;
+                color: @gray-2;
+            }
+        }
+
+        &.active {
+            border: 2px solid @gray-10;
+            background-color: @white;
+            height: fit-content;
+            white-space: nowrap;
+
+            > a {
+                visibility: visible;
+            }
+        }
+    }
+
+    &.anchor-top-left {
+        .action-menu {
+            top: -0.35rem;
+            right: initial;
+            left: -0.35rem;
+
+            > a {
+                padding: 0.3rem 1rem 0.3rem 2rem;
+            }
+        }
+    }
+
+    &.anchor-bottom-right {
+        .action-menu {
+            top: initial;
+            bottom: 0;
+        }
+    }
+
+    &.anchor-bottom-left {
+        .action-menu {
+            top: initial;
+            right: initial;
+            left: -0.35rem;
+            bottom: 0;
+
+            > a {
+                padding: 0.3rem 1rem 0.3rem 2rem;
+            }
+        }
+    }
+}
```

Comment:
Perhaps this component could need a `user-select: none;` to disable highlighting of component when double clicking.

Reply:
Yes!!! Thankyou!